diff --git a/jenkins/create-pull-secret/params.groovy b/jenkins/create-pull-secret/params.groovy
index e9433cf..85542d0 100644
--- a/jenkins/create-pull-secret/params.groovy
+++ b/jenkins/create-pull-secret/params.groovy
@@ -6,7 +6,7 @@ jobParameters = ([
             description: 'Кред с токеном jenkins сервис аккаунта'
         ),
         choice(
-            choices: ['dev','ift','psi','prod', 'rdt'],
+            choices: ['dev','ift','ift-behe4mln','psi','prod', 'rdt'],
             name: 'DEPLOY_STAND'
         ),
         [$class: 'CascadeChoiceParameter',
@@ -40,6 +40,10 @@ jobParameters = ([
 
                             return["a8ewkegg.k8s.delta.sbrf.ru"]
                         }
+                        if (DEPLOY_STAND == "ift-behe4mln") {
+
+                            return["behe4mln.k8s.delta.sbrf.ru"]
+                        }
                         if (DEPLOY_STAND == "prod") {
 
                             return["bdq2i0fy.k8s.omega.sbrf.ru"]
@@ -90,6 +94,10 @@ jobParameters = ([
                         if (DEPLOY_STAND == "ift") {
                             return["ci06618727-ozone"]
                         }
+                        if (DEPLOY_STAND == "ift-behe4mln") {
+                            // TODO: namespace второго кластера IFT (behe4mln)
+                            return[""]
+                        }
                         if (DEPLOY_STAND == "prod" || DEPLOY_STAND == "rdt") {
                             return["ci15207061-ozone"]
                         }
diff --git a/jenkins/deploy/params.groovy b/jenkins/deploy/params.groovy
index ccee606..95de141 100644
--- a/jenkins/deploy/params.groovy
+++ b/jenkins/deploy/params.groovy
@@ -6,7 +6,7 @@ jobParameters = ([
             description: 'Кред с токеном jenkins сервис аккаунта'
         ),
         choice(
-            choices: ['dev','ift','psi','prod', 'rdt'],
+            choices: ['dev','ift','ift-behe4mln','psi','prod', 'rdt'],
             name: 'DEPLOY_STAND'
         ),
         [$class: 'CascadeChoiceParameter',
@@ -40,6 +40,10 @@ jobParameters = ([
 
                             return["a8ewkegg.k8s.delta.sbrf.ru"]
                         }
+                        if (DEPLOY_STAND == "ift-behe4mln") {
+
+                            return["behe4mln.k8s.delta.sbrf.ru"]
+                        }
                         if (DEPLOY_STAND == "prod") {
 
                             return["bdq2i0fy.k8s.omega.sbrf.ru"]
@@ -90,6 +94,10 @@ jobParameters = ([
                         if (DEPLOY_STAND == "ift") {
                             return["ci06618727-ozone"]
                         }
+                        if (DEPLOY_STAND == "ift-behe4mln") {
+                            // TODO: namespace второго кластера IFT (behe4mln)
+                            return[""]
+                        }
                         if (DEPLOY_STAND == "prod" || DEPLOY_STAND == "rdt") {
                             return["ci15207061-ozone"]
                         }
